storage-proto/proto/transaction_by_addr.proto

```diff
@@ -70,6 +70,8 @@ message InstructionError {
     uint32 index = 1;
     InstructionErrorType error = 2;
     CustomError custom = 3;
+    optional uint32 inner_instruction_index = 4;
+    optional uint32 responsible_program_account_index = 5;
```

Comment:
@steviez is there something fancy I should be doing here? Creating an option (1 byte) and a `uint32` (4 bytes) just to store one byte of data sort of stinks. Could I instead hijack the remaining 24 bytes of `uint32 index = 1` to do this?

```
xxxxxxxx abbbbbbb bcdddddd dd------

x - existing 8 bytes for `index`
a - option flag for `inner_instruction_index`
b - new 8 bytes for `inner_instruction_index`
c - option flag for `responsible_program_account_index`
d - new 8 bytes for `responsible_program_account_index`
```

…then just figure it all out in `convert.rs`?